axum-macros/src/debug_handler/specializer.rs

```diff
@@ -88,8 +96,11 @@ impl Specializer {
         }
 
         Ok(Specializer {
+            item_fn,
             generic_params,
-            specializations,
+            specializations,            
```

Comment:
Trailing spaces again. Do you have `rustfmt` set up? It should remove these.

Reply:
yea i was pretty confused myself. there was something in specializer.rs that was breaking rustfmt, i just figured out what it was and changed it, so all of this soudl be handled now.